frb/builds/build_hosts.py

```diff
@@ -1760,41 +2085,836 @@ def build_host_200430(build_ppxf=False, build_photom=False, build_cigale=False,
+        #photom['VLT_FORS2_g_err'] = 0.5
+        #photom['VLT_FORS2_I'] = 22.0
+        #photom['VLT_FORS2_I_err'] = 0.4
+
+        #Merge and write
+        #photom = frbphotom.merge_photom_tables(photom, photom_file)
+        #photom.write(photom_file, format=frbphotom.table_format, overwrite=True)
+        #print("Wrote photometry to: {}".format(photom_file))
+    
+
+    # Load
+    #photom = Table.read(photom_file, format=frbphotom.table_format)
+    # Dust correct
+    #EBV = nebular.get_ebv(gal_coord)['meanValue']  # 0.061
+    #frbphotom.correct_photom_table(photom, EBV, 'HG')
+    # Parse
+    #host191228.parse_photom(photom, EBV=EBV)
+    
+
+    #print(host200906.photom.keys())
+    # CIGALE
+    
+    #cigale_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 'HG191228_CIGALE.fits')
+    #sfh_file = cigale_file.replace('CIGALE', 'CIGALE_SFH')
+    if build_cigale:
+        # Prep
+        #cut_photom = Table()
+        # Let's stick to DES only
+        #for key in host200906.photom.keys():
+         #   if 'DES' not in key:
+         #       continue
+         #   cut_photom[key] = [host200906.photom[key]]
+        # Run
+        #print(cut_photom)
+        print("Running cigale")
+        #cut_photom = None
+        # Run
+        #cigale.host_run(host20200120E,cigale_file=cigale_file) #, cut_photom=cut_photom, cigale_file=cigale_file)
+    # Parse
+    #warnings.warn("TURN 191228 on")
+    #host191228.parse_cigale(cigale_file, sfh_file=sfh_file)
+    
+
+    #warnings.warn("TURN 191228 photometry on")
+    #host191228.parse_cigale(cigale_file, sfh_file=sfh_file)
+
+    # PPXF
+    #ppxf_results_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 
+     #                                'HG191228_LRISr_ppxf.ecsv')
+    #spec_fit_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 
+     #                            'HG191228_LRISr_ppxf.fits')
+    if build_ppxf:
+        # LRIS??
+        meta, spectrum = host191228.get_metaspec(instr='LRISr', 
+                                                 specdb_file=os.path.join(
+                                                 os.getenv('SPECDB'), 
+                                                 'specDB_CRAFT.hdf5'))
+        # Correct for Galactic extinction
+        AV = EBV * 3.1  # RV
+        Al = extinction.ccm89(spectrum.wavelength.value, AV, 3.1)
+        # New spec
+        new_flux = spectrum.flux * 10**(Al/2.5)
+        new_sig = spectrum.sig * 10**(Al/2.5)
+        new_spec = XSpectrum1D.from_tuple((spectrum.wavelength, new_flux, new_sig))
+        #
+        R = 1607.
+
+        ppxf.run(new_spec, R, host191228.z, results_file=ppxf_results_file,
+                 spec_fit=spec_fit_file,
+                 atmos=[[3000., 5000.], [7580, 7750.]],
+                 gaps=[[6675., 6725.]], chk=True)
+    #host191228.parse_ppxf(ppxf_results_file)
+
+
+    # Derived quantities from Bhardwaj2021
+    host20200120E.derived['M_r'] = -19.78
```

Comment:
Did these come from CIGALE?
If so, provide the files and we should read
the values in from them.

i.e. no typing them in.

Reply:
Again not from CIGALE.